components/Nav/Nav.tsx

```diff
@@ -63,6 +64,16 @@ export const Nav = (): JSX.Element => {
                 <DrawerTitle data-testid="title">Gridiron Survivor</DrawerTitle>
               </DrawerHeader>
               <ul className="m-0 flex flex-col gap-4 p-0">
+                <li>
+                  <Link
+                    data-testid="league-link"
+                    href="/league/all"
+                    className="pb-3 text-base font-normal text-zinc-600"
+                    onClick={() => setOpen(false)}
+                  >
+                    Leagues
+                  </Link>
```

Comment:
@vazquezea96 great job! one thing to not is we could use the LinkCustom component that ryan built or FOR NOW use the styling from it.  I don't think the LinkCustom would work for you in this branch cause it doesn't allow onClick (which you need).  

My suggestion: take the styling for LinkCustom and use that here in the className so it is consistent with the other link.   We'll have to make a ticket to update it LATER once @ryandotfurrer's PR gets pulled in to develop.